pmaudit

```diff
@@ -167,16 +167,33 @@ def os_path_walk(top, func, arg):
         if stat.S_ISDIR(st.st_mode):
             os_path_walk(name, func, arg)
 
+def die(message):
+    """Report failure message and exit program with failure"""
+    print(message, file=sys.stderr)
+    sys.stderr.flush()
+    exit(1)
+
+def mkdir_p_or_die(new_dir):
```

Comment:
No, I still need to avoid the empty string.  Python returns the empty string if a file has not specified directory, and os.makedirs fails on the empty string.  So I have to specially store & check for the empty string every time on every caller, or write the check once in a called function.  I think it's cleaner to create a function so I only have to write the check once.

Reply:
Good point, I had not known that. I think os.makedirs() should handle the null string but who am I to challenge Guido et al? Never mind.